python/pylibcudf/pylibcudf/libcudf/strings/convert/convert_urls.pxd

```diff
@@ -8,7 +8,7 @@ from pylibcudf.libcudf.column.column_view cimport column_view
 cdef extern from "cudf/strings/convert/convert_urls.hpp" namespace \
         "cudf::strings" nogil:
     cdef unique_ptr[column] url_encode(
-        column_view input_col) except +
+        column_view input) except +
```

Comment:
@vyasr should we be marking these APIs all with `libcudf_exception_handler` as we go even if currently the API has no documented exception cases?

Reply:
I think that's probably a good best practice yeah.

Reply:
xref https://github.com/rapidsai/cudf/issues/17036 to follow up in the future